api/v1beta1/azurecluster_validation.go

```diff
@@ -498,3 +508,12 @@ func validateCloudProviderConfigOverrides(old, new *CloudProviderConfigOverrides
 	}
 	return allErrs
 }
+
+func validateOutboundIP(frontendIP FrontendIP) error {
+	outboundIP := net.ParseIP(frontendIP.PublicIP.OutboundIP)
+	_, _, err := net.ParseCIDR(frontendIP.PublicIP.OutboundIP)
+	if outboundIP == nil && err != nil {
+		return err
```

Comment:
can we wrap this into a error that says something about `outboundIP==nil` as well?